src/main/java/taxi/dao/CarDao.java

```diff
@@ -0,0 +1,17 @@
+package taxi.dao;
+
+import java.util.List;
+import java.util.Optional;
+import taxi.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    Optional<Car> get(Long id);
+
+    List<Car> getAll();
+
+    Car update(Car car);
+
+    boolean delete(Long id);
+}
```

Comment:
create this method please List<Car> getAllByDriver(Long driverId);

Reply:
done